skew/awsclient.py

```diff
@@ -127,7 +127,7 @@ def call(self, op_name, query=None, **kwargs):
                     done = True
                 except ClientError as e:
                     LOG.debug(e, kwargs)
-                    if 'Throttling' in str(e):
+                    if 'Throttling' in str(e) or 'SlowDown' in str(e):
```

Comment:
Assuming we might expand this down the line it might be easier to make a list of keywords we want to look for and use any. 

```
back_off_errors = ['Throttling', 'SlowDown']
if any(i in str(e) for i in back_off_errors):
```